src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Tov.cpp

```diff
@@ -79,20 +83,80 @@ class Observer {
   std::vector<double> log_enthalpy;
 };
 
+template <typename DataType>
+RelativisticEuler::Solutions::TovStar<
+    gr::Solutions::TovSolution>::RadialVariables<DataType>
+interior_solution(
+    const EquationsOfState::EquationOfState<true, 1>& equation_of_state,
+    const DataType& radius, const DataType& mass_inside_radius,
+    const DataType& log_specific_enthalpy,
+    const double log_lapse_at_outer_radius) noexcept {
+  RelativisticEuler::Solutions::TovStar<
+      gr::Solutions::TovSolution>::RadialVariables<DataType>
+      result(radius);
+  result.specific_enthalpy = Scalar<DataType>{exp(log_specific_enthalpy)};
+  result.rest_mass_density = equation_of_state.rest_mass_density_from_enthalpy(
+      result.specific_enthalpy);
+  result.pressure =
+      equation_of_state.pressure_from_density(result.rest_mass_density);
+  result.specific_internal_energy =
+      equation_of_state.specific_internal_energy_from_density(
+          result.rest_mass_density);
+  result.metric_time_potential =
+      log_lapse_at_outer_radius - log_specific_enthalpy;
+  result.dr_metric_time_potential =
+      (mass_inside_radius + 4.0 * M_PI * get(result.pressure) * cube(radius)) /
+      (radius * (radius - 2.0 * mass_inside_radius));
+  result.metric_radial_potential =
+      -0.5 * log(1.0 - 2.0 * mass_inside_radius / radius);
+  result.dr_metric_radial_potential =
+      (4.0 * M_PI * radius *
+           (get(result.specific_enthalpy) * get(result.rest_mass_density) -
+            get(result.pressure)) -
+       mass_inside_radius / square(radius)) /
+      (1.0 - 2.0 * mass_inside_radius / radius);
+  result.metric_angular_potential = make_with_value<DataType>(radius, 0.0);
```

Comment:
My understanding is that these "potentials" arise from the usual convention of writing down the metric in terms of exponentials of "potential" functions, e.g. -g_tt = e^(2\nu). If I'm not wrong, this "angular" potential would be one of the \mu's in Chandrasekhar's notation (the other one being the "radial" potential). At least in vacuum, and in areal coordinates, e^(2\mu_angular) = r^2 in Chandrasekhar's notation. Here it looks like you're factoring out a r^2, which is the usual convention for isotropic coords (e.g. in the Einstein Toolkit.) I think it' be helpful to document the relevant formulas, especially the ones related to the metric. Lindblom's paper (which our documentation points to) makes no reference to any "potential".

Also, why is introducing the angular potential needed? Do you expect to write an analogous solution in isotropic coords, for which the angular potential would be less trivial (namely angular potential = radial potential = (1 + 2M/r^2)^2)?

Reply:
Yes, that is why the angular potential is used